sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/AuditEvent.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Logic.Audit.Client.SampleLargeAuditEvents
+{
+    public class AuditEvent
+    {
+        public Guid Id { get; set; }
+
+        public Guid Name { get; set; }
+
+        public Guid Department { get; set; }
+
+        public Guid Age { get; set; }
+
+        public AuditEvent()
+        {
+            this.Id = Guid.NewGuid();
```

Comment:
why do we initialize it in constructor? can initialization be like the properties declared in sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/ProgramConfig.cs